src/styles/components/overrides/_results.scss

```
@@ -42,30 +42,98 @@
+    position: absolute;
+    inset-inline-start: 0;
+    inset-block-start: 50%;
+    transform: translate(0, -50%);
+    background: var(--border-light);
+    width: 2px;
+  }
 }
 
+.tools-toggle {
+  @include button-reset;
+  padding: $space-3 $space-4;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)